css/styles.css

```diff
@@ -67,8 +67,9 @@ ul li {
 
 #about-me > ul > li > ul > li > ul > li {
   background-color: white;
-  padding: 8px;
+  padding: 8px 3px;
   border-radius: 7px;
+  font-size: 12px;
 }
 
 .li-mid {
```

Comment:
- [ ] Could you please check the spacing at the bottom of your button? It appears to be incorrect according to the template. Could you kindly add the appropriate spacing?

   ### Figma design:
   ![image](https://github.com/flov3rh/portfolio/assets/107952724/3b48a1e1-8610-43a6-8026-5fe9aaca3be5)


   ### Your design:
   ![image](https://github.com/flov3rh/portfolio/assets/107952724/2916b3a5-435b-4d9c-980f-f95eefd69117)